src/lib.rs

```diff
@@ -105,6 +106,8 @@ pub use value::Value;
 #[cfg(feature = "http_client")]
 pub use client::AppConfigurationClientHttp;
 #[cfg(feature = "http_client")]
+pub use network::live_configuration::LiveConfigurationImpl;
+#[cfg(feature = "http_client")]
```

Comment:
why public?

Reply:
👍 Hidden. Anyway, we need to go through all the crate and try to make more things private.